### ISSUE TITLE
BITBOX.Script.classifyOutput() produces Error: Expected Hex, got String

### ISSUE THREAD

**Issue by sagech** [Bitcoin-com/bitbox-javascript-sdk#9]
```
let nullDataOutput = "OP_RETURN 877 8106baaeb4f1b3f7f2558b788be6cb62d76b5ea8eba64a05a509edd504d76fb7 f09fa496203120424348203d203536312e30322420f09f938820362e3533250a54697073203d206675656cefb88f";
let result = BITBOX.Script.classifyOutput(BITBOX.Script.fromASM(nullDataOutput));
console.log(result);
```

This block of code produces `Error: Expected Hex, got String "877".`

This OP_RETURN is from https://rest.bitcoin.com/v1/transaction/details/c4e93088b8658b9a02e52171b324512737e4ad9c7fb402ca6b9af1d8f9f038d8 output n=1

**Comment by sagech** [comment]
It seems like this comes from bitcoincashjs-lib

**Comment by cgcardona** [comment]
This is fixed in v1.5.9.
